js/analyses/calculateMetaDescriptionKeyword.js

```diff
@@ -0,0 +1,14 @@
+/** @module analyses/getMetaDescriptionKeyword */
```

Comment:
This module can be removed, it's now only a wrapper around `wordMatch`, which isn't particularly useful.